geometry_msgs/PointStamped[] locations
